### DISCUSSION TITLE
Do build artifacts of open source (public) projects count towards the storage limit?

### DISCUSSION THREAD

**Discussion by peterjuras** [community/community#26438] [Actions]
<p>Hi,</p>
<p>I wanted to ask if build artifacts of open source (public) projects count towards the storage limit or if they are free with unlimited storage?</p>
<p>Best regards,</p>
<p>Peter</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/peterjuras">@peterjuras</a> ,</p>
<p>GitHub Actions usage is free for public repositories. The build artifacts in public repo are not counted by storage limit.  </p>
<p>There is the official document for GitHub Actions billing: </p>
<p><a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions#about-billing-for-github-actions" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions#about-billing-for-github-actions</a></p>

**Comment by peterjuras** [comment]
<p>Thanks for the clarification, I saw the documentation and that is why I was asking. In particular the following quote from the article you shared …</p>
<p>"</p>
<p>The storage used by a repository is the total storage used by GitHub Actions artifacts and GitHub Packages. Your storage cost is the total usage for all repositories owned by your account.</p>
<p>"</p>
<p>… does not make it sound like storage for public repositories ist not counted (the “all repositories” sounds very specific). The other wording with “GitHub Actions usage is free for public repositories” is also not too re-assuring, because Storage is very often treated separately, for example also on the billing page.</p>
<p>So in total, thanks for reassuring that public build artifacts are free, but I think GitHub could make the documentation on this clearer.</p>

**Comment by Yanjingzhu** [comment]
<p>Oh, thank you for pointing this out. What a valuable improvement. I have submitted a request in our internal channel to update the billing document . Thank you for helping us build a better GitHub Actions.  </p>
